src/main/java/com/dreammaster/scripts/ScriptEFR.java

```diff
@@ -96,15 +107,71 @@ public void loadRecipes() {
                 getModItem(Minecraft.ID, "leather", 1, 0, missing),
                 bits,
                 new Object[] { "SSS", "HHH", "SSS", 'S', getModItem(Minecraft.ID, "string", 1, 0, missing), 'H',
                         getModItem(EtFuturumRequiem.ID, "rabbit_hide", 1, 0, missing) });
+        GTModHandler.addCraftingRecipe(
+                getModItem(EtFuturumRequiem.ID, "soul_lantern", 1, 0, missing),
+                bits,
+                new Object[] { "IGI", "PCP", "III", 'I', "plateIron", 'G', "dustInfusedWater", 'P',
+                        "paneGlassColorless", 'C', getModItem(EtFuturumRequiem.ID, "soul_torch", 1, 0, missing) });
+
+        GTModHandler.addCraftingRecipe(
+                getModItem(EtFuturumRequiem.ID, "target", 1, 0, missing),
+                bits,
+                new Object[] { "WTR", "CHD", "RTW", 'W', "blockWoolWhite", 'R', "blockWoolRed", 'T', "dustRedstone",
+                        'C', GTModHandler.getModItem(Minecraft.ID, "comparator", 1, 0), 'H',
+                        GTModHandler.getModItem(Minecraft.ID, "hay_block", 1, 0), 'D',
+                        GTModHandler.getModItem(ExtraUtilities.ID, "budoff", 1, 0) });
+
+        GTModHandler.addCraftingRecipe(
+                getModItem(EtFuturumRequiem.ID, "smoker", 1, 0, missing),
+                bits,
+                new Object[] { "PhP", "PFP", "SCS", 'P', "plateIron", 'F',
+                        getModItem(Minecraft.ID, "furnace", 1, 0, missing), 'S',
+                        getModItem(EtFuturumRequiem.ID, "smooth_stone", 1, 0, missing), 'C',
+                        getModItem(AdventureBackpack.ID, "blockCampFire", 1, 0, missing) });
+
+        GTModHandler.addCraftingRecipe(
+                getModItem(EtFuturumRequiem.ID, "end_crystal", 1, 0, missing),
+                bits,
+                new Object[] { "VGV", "VSV", "VQV", 'V',
+                        getModItem(AppliedEnergistics2.ID, "tile.BlockQuartzLamp", 1, 0, missing), 'G',
+                        getModItem(HardcoreEnderExpansion.ID, "living_matter", 1, 0, missing),
+                        'S', getModItem(DraconicEvolution.ID, "wyvernCore", 1, 0, missing), 'Q',
+                        ItemList.QuantumEye.get(1), });
+
+        GTModHandler.addCraftingRecipe(
+                getModItem(EtFuturumRequiem.ID, "smithing_table", 1, 0, missing),
+                bits,
+                new Object[] { "TT ", "GG ", "GG ", 'T',
+                        GTModHandler.getModItem(TinkerConstruct.ID, "CraftingSlab", 1, 5), 'G',
+                        GTModHandler.getModItem(Thaumcraft.ID, "blockWoodenDevice", 1, 6), });
+
+        GTValues.RA.stdBuilder().itemInputs(getModItem(Minecraft.ID, "magma_cream", 3, 0, missing))
+                .fluidInputs(new FluidStack(FluidRegistry.getFluid("lava"), 6000))
+                .itemOutputs(getModItem(EtFuturumRequiem.ID, "magma", 1, 0, missing)).duration(8 * SECONDS).eut(2)
+                .addTo(compressorRecipes);
```

Comment:
This seems like quite a bit of lava, especially if we decide to include this block in worldgen at any point